### ISSUE TITLE
gimp@2.10.30: decompress error

### ISSUE THREAD

**Issue by duncanc** [ScoopInstaller/Extras#7977]
The hash of gimp-2.10.30-setup.exe checks out but extraction fails.

#### Scoop Configuration

```json
{
    "lastupdate":  "2022-02-16T15:01:57.2120479+00:00",
    "SCOOP_REPO":  "https://github.com/lukesampson/scoop",
    "SCOOP_BRANCH":  "master"
}
```

**Comment by rashil2000** [comment]
Cannot reproduce.

What is the error? Note that it is a big program, so it will take a lot of time to extract.

**Comment by duncanc** [comment]
The error was occurring immediately as it started to extract. I worked out the previous version of GIMP was 2.10.22 and used `scoop install gimp@2.10.22` to install that version instead, which worked fine.

I did a bit more investigating and found that my version of innounp was not up-to-date, and this was causing the error. It appears that in version 2.10.30, GIMP updated its Inno Setup version to 6.1.0, which requires innounp 0.50 or above to extract (I had 0.49).

I believe this means that GIMP's manifest should be updated to include an install-time dependency on innounp 0.50+, or something of that nature (I don't know all the details of how manifests work).
